details_land.md

```diff
@@ -76,14 +76,10 @@ For example, the current data set has very high-yielding wheat in the panhandle
 
 ## Regional Production and Comparative Advantage
 
-In its determination of the economic allocation of crop production and
-land use across regions of the globe, GCAM follows the well-known but
-often misunderstood basic economic principle of comparative
-advantage. The theory of comparative advantage was identified and
-explained by David Ricardo in the early 19th Century (Ricardo, 1817)
-to explain patterns in the study of international production and trade
-(Samuelson and Nordhaus, 2004). In simplest terms, countries or
-regions will produce more of what they are better at and import more
+In its determination of the economic allocation of crop production and land use across regions of the globe, GCAM follows the well-known but
```

Comment:
I think this paragraph should all be combined to one line, rather than having line breaks interspersed throughout spread over multiple?  I think web browsers deal with the line wrapping dynamically as needed.

Reply:
fixed